src/utils/health-check.ts

```diff
@@ -39,14 +45,21 @@ export async function healthCheck(peerId?: string) {
         profile.peerId !== formatFullPeerString(peerId, process.env.DOMAIN)
       ) {
         logCallout(async () => {
-          archLogger.error("Peer ID on profile does not match local Peer Id!\n", true);
-          archLogger.error("Please update your profile \n", true);
-          archLogger.error("Your archaeologist will not appear in the embalmer webapp\n", true);
+          archLogger.error("Peer ID on profile does not match local Peer Id!\n", {
+            logTimestamp: true,
+          });
+          archLogger.error("Please update your profile \n", { logTimestamp: true });
+          archLogger.error("Your archaeologist will not appear in the embalmer webapp\n", {
+            logTimestamp: true,
+          });
           archLogger.warn(`Local Peer ID: ${process.env.DOMAIN}:${peerId}`, true);
           archLogger.warn(`Profile Peer ID: ${profile.peerId}`, true);
         });
 
-        // TODO -- add notification once notifications are setup
+        await notifyUser(
+          "There is a problem with your archaeologist profile. Please check the logs on your node for more information."
+        );
```

Comment:
We could send the actual information of the error in the notification, since we have that info in the context here (peerID does not match)

Reply:
Ah yes, you're right